src/components/modules/monthly-review/ui/MonthlyReviewDashboard.tsx

```
@@ -0,0 +1,419 @@
+      return current.horizon.dailyTrend.map((item) => ({
+        date: item.date.slice(8, 10), // just day number
+        transactions: item.txCount,
+      }));
+    }
+
+    // Fallback daily mockup spread over a month for clean visuals
+    const daysInMonth = 30;
+    const baseTx = Math.round(current.horizon.txCount / daysInMonth);
+    return Array.from({ length: daysInMonth }).map((_, i) => ({
+      date: String(i + 1),
+      transactions: Math.round(baseTx * (0.85 + Math.random() * 0.3)),
```

ESLint

[error] Error: Cannot call impure function during render

`Math.random` is an impure function. Calling an impure function can produce unstable results that update unpredictably when the component happens to re-render. (https://react.dev/reference/rules/components-and-hooks-must-be-pure#components-and-hooks-must-be-idempotent).

/home/jailuser/git/src/components/modules/monthly-review/ui/MonthlyReviewDashboard.tsx:69:49
  67 |     return Array.from({ length: daysInMonth }).map((_, i) => ({
  68 |       date: String(i + 1),
> 69 |       transactions: Math.round(baseTx * (0.85 + Math.random() * 0.3)),
     |                                                 ^^^^^^^^^^^^^ Cannot call impure function
  70 |     }));
  71 |   }, [current, period]);
  72 |

(react-hooks/purity)